### PR TITLE
Added some workflows

### PR DESCRIPTION
**Pull request by asmiagarwal**
## Describe your changes

I have added some workflows (github actions).

1. The `Labeler` github action labels the PR with hacktoberfest-accepted when 1 approving review is done.
2. The `Community` github action greets user when they open a PR or Issue.

## Screenshots - If Any (Optional)

image

## Issue ticket number and link - If Any

No

## Checklist before requesting a review

- [x] I have performed a self-review of my code.

- [x] Followed the repository's [Contributing Guidelines](https://github.com/dvstechlabs/Noteslify/blob/main/CONTRIBUTING.md).

- [x] I ran the app and tested it locally to verify that it works as expected.

- [x] I have starred the repository.

## Additional Information (Optional)

cc @devarshishimpi and @MrKrishnaAgarwal

### CODE REVIEW

**Review by MrKrishnaAgarwal — APPROVED** [review]
Looks good!
Thanks for adding these amazing github actions 🎉

**Comment by MrKrishnaAgarwal** [comment]
@devarshishimpi Please take a look here too!

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+name: Label approved pull requests
+jobs:
+  labelWhenApproved:
+    name: Label when approved
+    runs-on: ubuntu-latest
+    steps:
+    - name: Label when approved
+      uses: pullreminders/label-when-approved-action@master
+      env:
+        APPROVALS: "1"
```
Comment (MrKrishnaAgarwal) [inline .github/workflows/labeler.yml:11]:
```suggestion
        APPROVALS: "2"
```

You can make here to 2 if @devarshishimpi wants 🤔

**Review by devarshishimpi — APPROVED** [review]

File: .github/workflows/labeler.yml (hunk 1 above)
Comment (devarshishimpi) [reply .github/workflows/labeler.yml:11]: I'll do that

**Review by devarshishimpi — APPROVED** [review]
LGTM!! Thanks For Contributing!!

**Pull request merged**